src/Pages/LandingHomePage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import '../App.css';
+import '../Styles/LandingPage.css'
+import {Button} from '../Components/LoginButton/LoginButton';
+import Navbar from '../Components/Navbar/Navbar'
+
+function LandingPage() {
+  React.useEffect(() => {
+    document.body.style.overflow = 'hidden';
```

Comment:
put in css file

Reply:
its a react useEffect, cant put in a css file

Reply:
you are defining a style in said useEffect which can be put in a css file

Reply:
which is also being removed, which last time I checked, isnt something a css file can do. Maybe read it

Reply:
I can read just fine champ.
Quit talking like you're hot shit. You came up with a lazy solution without a second though.

```
.landing-page-overflow-fix{
    height: 100vh;
    overflow: hidden;
}

.page {
    display:flex;
    height: 100%;
    background: url('../images/hockeybackground.jpeg');
    background-size: cover;
}
```
```
 <div className="landing-page-overflow-fix">
            <Navbar />
            <div className="page">
...
            </div>
</div>
```

Reply:
my bad